global.css

```diff
@@ -176,7 +176,6 @@ body {
 .footer {
   width: 100vw;
   text-align: center;
-  position: fixed;
```

Comment:
did you check if removing this position:fixed don't impact other pages

Reply:
Yes I've checked